core/src/main/java/com/linecorp/armeria/common/RequestContext.java

```diff
@@ -276,6 +209,114 @@ default EventLoop contextAwareEventLoop() {
         return new RequestContextAwareEventLoop(this, eventLoop());
     }
 
+    /**
+     * Pushes the specified context to the thread-local stack. To pop the context from the stack, call
+     * {@link SafeCloseable#close()}, which can be done using a {@code try-with-resources} block.
+     *
+     * @see #push()
+     * @deprecated Use {@link #push()} instead.
```

Comment:
- Please remove 'instead' for consistency. (Originally suggested by @anuraaga)
- Probably OK to remove the `@see` tag because it's mentioned by `@deprecated`?